usecases/src/main/java/com/architectcoders/usecases/GetCharacters.kt

```diff
@@ -3,6 +3,14 @@ package com.architectcoders.usecases
 import com.architectcoders.data.repository.CharacterRepository
 import com.architectcoders.domain.characters.Characters
 
-class GetCharacters(private val characterRepository: CharacterRepository) {
-    suspend fun invoke(offset: Int): Characters = characterRepository.getCharactersRemote(offset)
+class GetCharacters(private val characterRepository: CharacterRepository): IUseCase<Characters,Int>{
+    override suspend fun  invoke(params: Int): Characters  =
+           characterRepository.getCharactersRemote(params)
+
+
+
+
+    // suspend fun invoke(offset: Int): Characters = characterRepository.getCharactersRemote(offset)
```

Comment:
@jujoru could you remove the commented code?

Reply:
Yes ¡¡¡ thanks.